### PR TITLE
Added cute color to the building outlines

### PR DESCRIPTION
**Pull request by FGRCL**

### CODE REVIEW

**Review by jonZlotnik — COMMENTED** [review]
I pulled and I don't see any building highlights. Do I need to activate them or something?

**Comment by FGRCL** [comment]
You first need to run the Migrator activity to populate the database. I know that's not actually what a migration is but I didn't have a better name.

File: app/src/main/java/com/example/campusguide/utils/BuildingHighlights.kt [hunk 1]
```diff
@@ -33,4 +37,12 @@ class BuildingHighlights(private val googleMap: GoogleMap) {
...
         return latLngs
     }
+
+    private fun applyPolygonStyle(polygon: PolygonOptions){
+        val primaryColor = ContextCompat.getColor(context, R.color.colorPrimary)
+        val darkColor = ContextCompat.getColor(context, R.color.colorPrimaryDark)
+        polygon.strokeWidth(3.5F)
+        polygon.strokeColor(darkColor)
+        polygon.fillColor(ColorUtils.setAlphaComponent(primaryColor, 40))
+    }
```
Comment (jun-loke) [inline app/src/main/java/com/example/campusguide/utils/BuildingHighlights.kt:47]: Could it be worth it to define the z index?

**Review by jun-loke — COMMENTED** [review]
Looks real cute! Added discussion comment for polygons style

File: app/src/main/java/com/example/campusguide/utils/BuildingHighlights.kt (hunk 1 above)
Comment (FGRCL) [reply app/src/main/java/com/example/campusguide/utils/BuildingHighlights.kt:47]: Yes, maybe once we have more stuff on the map overlay

**Review by jun-loke — APPROVED** [review]
All good

**Pull request merged**